lib/gemcutter/middleware/hostess.rb

```diff
@@ -1,51 +1,52 @@
 module Gemcutter::Middleware
-  class Hostess < Rack::Static
```

Comment:
I would suggest not to change this. I understand the reasoning of https://rubystyle.guide#namespace-definition, however, I don't think these classes are offenders and rarely see any change.
these classes have created issues with rails autoloader in the past.

Reply:
oh that's why I was getting loading errors, will disable this rule

Reply:
fixed in c7d05c845b7e903d27f12f918ae9263447f3b492